src/Components/App/App.js

```diff
@@ -67,52 +69,46 @@ const App = () => {
     setCurrentUser(id);
   }
 
-  const filterChosenVendor = () => {
-
-  }
   const resestResults = (event) => {
     event.preventDefault();
     fetchData()
   }
-const fetchData = () => {
-  const thisData = mockData.data.map((data) => {
-    return data.attributes
-  })
-  setVendors( thisData)
-  // this is going to fetch the data and then set state but then also reset isLoading to false
-}
+
+  const fetchData = () => {
+    const thisData = mockData.data.map((data) => {
+      return data.attributes
+    })
+    setVendors( thisData)
+    // this is going to fetch the data and then set state but then also reset isLoading to false
+  }
+
   useEffect(() => {
     fetchData();
 // call in mock data somehow and establish it as the state of vendors
   }, [])
 
   return(
-    <Routes>
-      <Route path="/" element={
+    <div>
+      <Header togView={toggleView} currentUser={currentUser}/>
+      <Routes>
+        <Route path="/" element={
           <div>
-            <Header togView={toggleView} currentUser={currentUser}/>
             <Search vendors={vendors} search={searchResults} reset={resestResults} allSearch={searchButtons}/>
-            <Results vendors={vendors} remFav={removeFav} addFav={favTruck} favorites={favorites} dist={importDistance}/>
+            <Results vendors={vendors} remFav={removeFav} addFav={favTruck} favorites={favorites} />
           </div>
-      } />
-      <Route path="/vendor/:id" element={
-          <div>
-            <Header togView={toggleView} />
-            {/* <TruckDetails /> */}
-            {/* //params.match, need to pass down the id that is clicked as a prop to truck details to element specfic truck info */}
-          </div>
-      } />
+        } />
+      <Route path="/vendor/:id" element={<TruckDetails vendors={vendors}/>}
```

Comment:
This is such a bizzare change that Router made for some reason, i can see why this was annoying to work wiht but phenomenal job working through it to find a solution!